packages/perseus/src/widgets/interactive-graphs/graphs/linear-system.test.tsx

```diff
@@ -0,0 +1,298 @@
+    let userEvent: UserEvent;
+    beforeEach(() => {
+        userEvent = userEventLib.setup({
+            advanceTimers: jest.advanceTimersByTime,
+        });
+        jest.spyOn(Dependencies, "getDependencies").mockReturnValue(
+            testDependencies,
+        );
+    });
+
+    test("should have aria label and describedby for overall linear system graph", () => {
+        // Arrange
+        render(
+            <MafsGraph {...baseMafsGraphProps} state={baseLinearSystemState} />,
+        );
+
+        // Act
+        const linearSystemGraph = screen.getByLabelText(
+            "Two lines on a coordinate plane.",
+        );
+
+        // Assert
+        expect(linearSystemGraph).toBeInTheDocument();
+        expect(linearSystemGraph).toHaveAttribute(
+            "aria-describedby",
+            ":r1:-line1-points :r1:-line1-intercept :r1:-line1-slope :r1:-line2-points :r1:-line2-intercept :r1:-line2-slope",
+        );
+    });
+
+    // Test each line in the linear system graph separately.
+    describe.each`
+        lineSequence
+        ${1}
+        ${2}
+    `(`Line $lineSequence`, ({lineSequence}) => {
+        test.each`
+            case                         | coords              | interceptDescription
+            ${"origin intercept"}        | ${[[1, 1], [2, 2]]} | ${"The line crosses the x and y axes at the graph's origin."}
+            ${"both x and y intercepts"} | ${[[4, 4], [7, 1]]} | ${"The line crosses the X-axis at 8 comma 0 and the Y-axis at 0 comma 8."}
+            ${"x intercept only"}        | ${[[5, 5], [5, 2]]} | ${"The line crosses the X-axis at 5 comma 0."}
+            ${"y intercept only"}        | ${[[5, 5], [2, 5]]} | ${"The line crosses the Y-axis at 0 comma 5."}
+        `(
+            "slope description should include slope info for $case",
+            ({coords, interceptDescription}) => {
+                // Arrange
+                const newCoords = [...baseLinearSystemState.coords];
+                newCoords[lineSequence - 1] = coords;
+
+                render(
+                    <MafsGraph
+                        {...baseMafsGraphProps}
+                        state={{
+                            ...baseLinearSystemState,
+                            coords: newCoords,
+                        }}
+                    />,
+                );
+
+                // Act
+                const linearSystemGraph =
+                    screen.getByLabelText(overallGraphLabel);
+
+                // Assert
+                expect(linearSystemGraph).toHaveTextContent(
+                    interceptDescription,
+                );
+            },
+        );
+
+        test.each`
+            case                 | coords              | slopeDescription
+            ${"positive slope"}  | ${[[1, 1], [3, 3]]} | ${`Its slope increases from left to right.`}
+            ${"negative slope"}  | ${[[3, 3], [1, 6]]} | ${`Its slope decreases from left to right.`}
```

Comment:
Maybe we should get a content creator to weigh in on this wording. Strictly speaking, I don't think the _slope_ increases or decreases; the slope is constant and it's the y-coordinate that increases or decreases.

Maybe "Its slope is positive/negative/zero/undefined" would be a good way to phrase this? In any case, I'd defer to Charlie or Nick.

Reply:
That's a good idea. Personally, I don't think that should be a blocker, I think we should update the wording when we do our strings pass before creating translation tickets.